app/models/accounting/customer.rb

```diff
@@ -25,11 +25,18 @@ def self.create_or_update_from_qb_object!(qb_object_type:, qb_object:)
 
   # The quickbooks-ruby gem does not implement a helper method for _id like account or class,
   # and in qb api line items need entity, not customer_id.
-  def reference
+  def entity
```

Comment:
This used to be called 'ref' but it actually builds an entity, and that's what QB api wants for a je line item. QB needed a reference for a check line item. So that's why there's both methods now.